src/renderer/common/components/dialog/publicationInfos/publicationInfoA11y.tsx

```diff
@@ -207,33 +209,33 @@ export class PublicationInfoA11y extends React.Component<IProps, IState> {
-            <ul>
-                {AccessModeSufficient ? AccessModeSufficient : <></>}
-                {AccessibilityFeatureIsprintPageNumber ? AccessibilityFeatureIsprintPageNumber : <></>}
-                {AccessibilityFeatureIsDisplayTransformability ? AccessibilityFeatureIsDisplayTransformability : <></>}
-                {AccessibilityFeatureIsSynchronizedAudioText ? AccessibilityFeatureIsSynchronizedAudioText : <></>}
-                {AccessibilityHazard ? AccessibilityHazard : <></>}
-            </ul>
-            : <></>}
-            {(a11y_certifiedBy || a11y_certifierCredential || AccessibilityFeature || AccessibilityConformsTo || AccessibilityConformanceReport || AccessibilitySummary) ?
-            <div>
-                <details>
-                    <summary>{__("publication.accessibility.moreInformation")}</summary>
-                    <ul>
-                        {AccessibilityFeature ? AccessibilityFeature : <></>}
-                        {AccessibilityConformsTo ? AccessibilityConformsTo : <></>}
-                        {AccessibilityConformanceReport ? AccessibilityConformanceReport : <></>}
-                        {a11y_certifiedBy ? a11y_certifiedBy.map((v, i) => (<li key={`kCertBy${i}`} style={{listStyleType: "none"}}>({v})</li>)) : <></>}
-                        {a11y_certifierCredential ? a11y_certifierCredential.map((v, i) => (<li key={`kCertCred${i}`} style={{listStyleType: "none"}}>({v})</li>)) : <></>}
-                        {AccessibilitySummary}
+            <>
+                {(AccessModeSufficient || AccessibilityHazard || AccessibilityFeatureIsprintPageNumber || AccessibilityFeatureIsSynchronizedAudioText || AccessibilityFeatureIsDisplayTransformability) ?
+                    <ul className={stylePublication.accessibility_infos_left}>
+                        {AccessModeSufficient ? AccessModeSufficient : <></>}
+                        {AccessibilityFeatureIsprintPageNumber ? AccessibilityFeatureIsprintPageNumber : <></>}
+                        {AccessibilityFeatureIsDisplayTransformability ? AccessibilityFeatureIsDisplayTransformability : <></>}
+                        {AccessibilityFeatureIsSynchronizedAudioText ? AccessibilityFeatureIsSynchronizedAudioText : <></>}
+                        {AccessibilityHazard ? AccessibilityHazard : <></>}
                     </ul>
-                </details>
-            </div>
-            : <></>}
-        </>
-        : <p>{__("publication.accessibility.noA11y")}</p>;
+                    : <></>}
+                {(a11y_certifiedBy || a11y_certifierCredential || AccessibilityFeature || AccessibilityConformsTo || AccessibilityConformanceReport || AccessibilitySummary) ?
+                    <div className={stylePublication.accessibility_infos_right}>
+                        {/* <details> */}
```

Comment:
@danielweck Just a question : there are no accessibility issue to remove details and summary html elements ?

Reply:
The HTML summary/detail elements provide built-in interaction behaviour and semantics, compared with div, and should be easier to style too. Better avoid generic HTML containers like div when there is a semantically-stronger alternative in the web standard.